usaspending_api/common/helpers/business_categories_helper.py

```diff
@@ -0,0 +1,143 @@
+# Dictionary of Business Categories that pair them with their human readable name
```

Comment:
Dumb question.... would a dictionary with the field name as the key and the display name as the value not work here?

```
{
    'category_business': 'Category Business',
    'small_business': 'Small Business',
    ...
}
```

Seems like it might simplify the code a bit.

Reply:
That was the direction I first went, but I wasn't sure if we would ever want to look them up by display name. Figured having a list of dictionaries would allow a lookup either direction. If it will rarely happen then I agree it would be much simpler to go with a dictionary where the key is the field_name and the value is the display_name and can make that change.

Reply:
I mean, you can still look up in the other direction in a dictionary pretty much the same way you're doing it here.  And if it's something you do regularly, you can just make an inverse dictionary based on the original dictionary.